samples/DNXLibrary/project.json

```diff
@@ -9,13 +9,12 @@
     "dependencies": {
       "xunit": "2.1.0-rc1-build3168",
       "xunit.runner.dnx": "2.1.0-beta5-build169",
-      "Microsoft.DotNet.xunit.performance": "99.99.99-dev"
+      "Microsoft.DotNet.xunit.performance": "1.0.0-alpha-build0015"
```

Comment:
As above, this could/should be the 99.99.99-dev version with automatic replacement in the .MSBuild file.
The idea is that, while developing locally that's the version you'll use. When it comes to release the NuGet packages, then we'll use the real semantic version.
Note that this is the same technique that xunit itself uses.